src/client/app/containers/import.ts

```diff
@@ -119,17 +120,22 @@ export class Import {
 
     strings = Strings();
 
-    constructor(private _store: Store<fromRoot.State>) {
+    constructor(private _store: Store<fromRoot.State>, private _request: Request) {
```

Comment:
Does our Import now support a view URL?  It should.  Please add it to the testing scenarios list, too...